source/includes/api/_dataset.md

```diff
@@ -868,63 +1109,63 @@ curl -X PATCH https://api.resourcewatch.org/v1/dataset/<dataset-id> \
+vocabularies        |                                                    Cluster of tags                                                     |  Object | `{"vocabularyOne": {"tags": [<tags>]},"vocabularyTwo": {"tags": [<tags>]}}` |
+widgetRelevantProps |                                          Group of relevant props of a widget                                           |   Array | Any Text |
+layerRelevantProps  |                                           Group of relevant props of a layer                                           |   Array | Any Text |
+subscribable        | Available dataset queries for subscriptions parameters accepted: `{{begin}}` for date begin and `{{end}}` for date end |  Object | `{" <queryname>": "<querybodytemplate>"}` |
+env                 | The environment you want the dataset to be accessible on. Defaults to production |  Text | `staging`, `production` or `preproduction` |
+
+*Note 1: Updating a dataset requires authentication.*
+
+Note 2: When updating the `env` property of a dataset the change will cascade down to all associated layer and widget entities.
 
 ## Deleting a Dataset
 
-**When a dataset is deleted the user's applications that were present on the dataset will be removed from it. If this results in a dataset without applications, the dataset itself will be then deleted.**
+> Example request for deleting a dataset
 
-Datasets can be deleted either by with role ADMIN or by the user with role MANAGER that created them.
+```shell
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/<dataset-id> \
+-H "Authorization: Bearer <your-token>"
+-H "Content-Type: application/json"
+```
 
-In order to perform this operation, the following conditions must be met:
+In order to delete a dataset, the following conditions must be met:
 
 - the dataset's `protected` property must be set to `false`.
 - the user must be logged in and belong to the same application as the dataset
 - the user must match one of the following:
   - have role `ADMIN`
   - have role `MANAGER` and be the dataset's owner (through the `userId` field of the dataset)
 
-Note that the deletion process cascades in a non-atomic way; deleting a dataset will also attempt to remove all the layers, widgets, vocabularies, related knowledge graph elements, and metadata entities associated with it. However, if one of these deletes fails (ie. attempting to delete a protected layer will fail), the dataset will still be deleted, and those resources will be left orphaned.
-
-<aside class="warning">
-Deleting a dataset removes it permanently! It is recommended that you save a local copy before doing so.
-</aside>
+**When a dataset is deleted, the user's applications that were present on the dataset will be removed from it. If this results in a dataset without applications, the dataset itself will be then deleted.**
```

Comment:
I'm confused by this. Are we talking about how _other_ datasets might get deleted?